backend/spec/features/admin/orders/cancelling_and_resuming_spec.rb

```diff
@@ -13,8 +13,8 @@
   let(:order) do
     order = create(:order)
     order.update_columns({
-      :state => 'complete',
-      :completed_at => Time.now
+      state: 'complete',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.